RNTester/src/RNTesterExampleList.re

```diff
@@ -21,37 +21,36 @@ let styles =
       }
     );
 
-include
-  SectionList.CreateComponent {
-    type item = ExampleList.item;
-  };
-
 let component = ReasonReact.statelessComponent "RNTesterExampleList";
 
-let make = {
-  let renderItem onPress {item} =>
-    <TouchableHighlight onPress=(fun () => onPress item)>
-      <View style=styles##row>
-        <Text style=styles##rowTitleText> (ReasonReact.stringToElement item.title) </Text>
-        <Text style=styles##rowDetailText> (ReasonReact.stringToElement item.description) </Text>
-      </View>
-    </TouchableHighlight>;
-  let itemSeparatorComponent {highlighted} =>
-    <View
-      style=(
-              if highlighted {
-                styles##separatorHighlighted
-              } else {
-                styles##separator
-              }
-            )
-    />;
-  fun ::components ::onPress _children => {
+let renderItem onPress =>
+  SectionList.renderItem (
+    fun {item} =>
+      <TouchableHighlight onPress=(fun () => onPress item)>
+        <View style=styles##row>
+          <Text style=styles##rowTitleText>
+            (ReasonReact.stringToElement item.ExampleList.title)
+          </Text>
+          <Text style=styles##rowDetailText> (ReasonReact.stringToElement item.description) </Text>
+        </View>
+      </TouchableHighlight>
+  );
+
+let itemSeparatorComponent =
+  SectionList.separatorComponent (
```

Comment:
I don't think so, they are there because of the conversion problematic I tried to explain above :D
As you know, for good performance in React functions as props should be memoized. But we did conversions from record to jsObject in render, i.e. created a new function each time! My idea with these functions was to make this conversion step more explicit and **outside** render, so that things could get memoized if necessary.